src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    public static void getStatistic(String fromFileName, String toFileName) {
+
+        File fileRead = new File(fromFileName);
+
+        int sumSupply = 0;
+        int sumBuy = 0;
+
+        try (BufferedReader reader = new BufferedReader(
+                new FileReader(fileRead))) {
+
+            StringBuilder stringBuilder = new StringBuilder();
+
+            // Read from file
+            String value = reader.readLine();
+            while (value != null) {
+                stringBuilder.append(value).append(" ");
+                value = reader.readLine();
+            }
```

Comment:
we can close this try block after we have read all of the strings to keep the opened connection to the file as short as possible